lndr-backend/data/lndr-server.config.j2

```diff
@@ -49,13 +49,13 @@ urban-airship {
   secret = "{{URBAN_AIRSHIP_SECRET}}"
 }
 
-
 heartbeat-interval = {{HEARTBEAT_INTERVAL}}
 
 aws {
   photo-bucket = "{{S3_PHOTO_BUCKET}}"
   access-key-id = "{{AWS_ACCESS_KEY_ID}}"
   secret-access-key= "{{AWS_SECRET_ACCESS_KEY}}"
+  notification-api-key= "{{NOTIFICATION_API_KEY}}"
```

Comment:
🎨Missing a space here before the `=`. Same goes for L57, which it looks like we overlooked in an earlier PR.